src/components/atoms/SingleMember.jsx

```diff
@@ -8,38 +8,38 @@ import Back from './ProfileBack'
 
 const useStyles = makeStyles({
   profileCard: {
-  padding: "1em",
-  width: "280px",
-  border: "10px solid white",
-  margin: "auto 10px",
-  paddingRight: "0em",
-  paddingBottom: "0em",
-  color: "white",
-  background: "linear-gradient(180deg, rgb(129, 8, 156) 1.99%, #662AAF 50.9%), rgb(112, 35, 129)",
-  mixBlendMode: "multiply",
+    padding: "1em",
+    width: "280px",
+    border: "10px solid white",
+    margin: "auto 10px",
+    paddingRight: "0em",
+    paddingBottom: "0em",
+    color: "white",
+    background: "linear-gradient(180deg, rgb(129, 8, 156) 1.99%, #662AAF 50.9%), rgb(112, 35, 129)",
+    mixBlendMode: "multiply",
   }
 })
 
-function SingleMember (props) {
-  const { favoriteShow, favoriteMistake, isFlipped} = props.profile
-  const { handleFlip } = props  
+function SingleMember(props) {
+  const [isFlipped, setIsFlipped] = useState(false);
```

Comment:
Since the flip state is only used right here, so I moved the state down to this component.